shell/pages/auth/login.vue

```diff
@@ -165,7 +165,7 @@ export default {
     },
 
     kubectlCmd() {
-      return "kubectl get secret --namespace cattle-system bootstrap-secret -o go-template='{{.data.bootstrapPassword|base64decode}}{{\"\\n\"}}'";
+      return "kubectl get secret --namespace cattle-system rancher-admin-secret -o jsonpath='{.data.password}' | base64 --decode; echo";
```

Comment:
Please follow the normal guidelines when editing their code.

Reply:
Done.

Reply:
Please moved the commented out code inside the Verrazzano change markers.

Reply:
Got it. Done.